doc/classes/Input.xml

```diff
@@ -30,6 +30,13 @@
 				If the specified action is already pressed, this will release it.
 			</description>
 		</method>
+		<method name="add_arbitrary_joy">
+			<return type="int" />
+			<param index="0" name="name" type="String" />
+			<description>
+				Returns the device index of a newly created arbitrary joypad.
```

Comment:
The description should mention what arbitrary joypads can be used for, and how to set them up from a script or extension. You could link to a third-party resource if needed, but it's important not to leave people "in the blind" so to speak :slightly_smiling_face:

Reply:
I think I've got a satisfactory implementation. It doesn't include *where* to get the processed data from, but I think that's a bit outside the scope of what the example needs to show

Reply:
Added a CSharp section to the example!